cdk/cdk.sh

```diff
@@ -1,6 +1,20 @@
-for dir in ../cfn-resources/*;
+# This shell script can be use to generate one or multiple L1 Constrcuts from CFN resources.
+# How to execute:
+#   For all resources without arguments  :  ./cdk.sh
+#   For specific resource with arguments :  ./cdk/sh "auditing cluster"  
```

Comment:
I assume this should be just `./cdk.sh "auditing"` OR `./cdk.sh "cluster"`? Other than this LGTM

Reply:
@sangramPeerI can you resolve this? Then I can merge